Makefile

```diff
@@ -47,6 +48,9 @@ closure: build/j2me.js aot
 shumway: $(SHUMWAY_SRCS)
 	node tools/tsc.js --sourcemap --target ES5 shumway/references.ts --out build/shumway.js
 
+build:
```

Comment:
We already have a directory named `build` when do `make j2me`. To avoid confliction, we need to choose a different target name. With the name `build`, `make build` won't update `config/build.js` unless we recreate the `build` directory.

Also the target should be added to the .PHONY list